solutions/swb-app/src/userRoutes.ts

```diff
@@ -52,8 +54,23 @@ export function setUpUserRoutes(router: Router, userService: UserManagementServi
   router.get(
     '/users',
     wrapAsync(async (req: Request, res: Response) => {
-      const users = await userService.listUsers();
-      res.status(200).json({ users });
+      const validatedRequest = validateAndParse<ListUsersRequest>(ListUsersRequestParser, {
+        ...req.query
+      });
+      try {
+        const users = await userService.listUsers(validatedRequest);
+        res.status(200).json({ users });
+      } catch (e) {
+        if (Boom.isBoom(e)) {
+          throw e;
+        }
+
+        if (isInvalidPaginationTokenError(e)) {
```

Comment:
Nitpick: It's outside the scope of this PR, but since we use this code segment repeatedly (line 68 to 70), we should move it to `errorHandler.ts`. Then we add that middleware in `generateRouter`. That way our code will by more DRY and we won't have to add line 68 to 70 for each paginated API.